scripts/soroban-settings/SorobanSettingsUpgrade.py

```diff
@@ -31,25 +31,85 @@
 
 # hardcode the upgrade you want to do here.
 def get_upgrade_set():
-    state_exp_settings = StateExpirationSettings(max_entry_expiration=Uint32(6312000),
-                                                 min_temp_entry_expiration=Uint32(
-                                                     16),
-                                                 min_persistent_entry_expiration=Uint32(
-                                                     86400),
+
+    max_contract_size = Uint32(65536) # 64 kb
+
+    contract_size_upgrade_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_MAX_SIZE_BYTES,
+        contract_max_size_bytes = max_contract_size)   
+        
+    compute_settings = ConfigSettingContractComputeV0(ledger_max_instructions=Int64(1000000000),
+                                                 tx_max_instructions=Int64(100000000),
+                                                 fee_rate_per_instructions_increment=Int64(100),
+                                                 tx_memory_limit=Uint32(41943040)) # 40 mb
+
+    compute_upgrade_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_COMPUTE_V0,
+        contract_compute = compute_settings)    
+    
+    contract_ledger_cost_settings = ConfigSettingContractLedgerCostV0(ledger_max_read_ledger_entries=Uint32(400),
+                                                 ledger_max_read_bytes=Uint32(2048000),
```

Comment:
I would suggest expressing per-ledger limits in terms of per-transaction limits, i.e. set this programmatically to `NUM_MAX_TXS_PER_LEDGER * <tx_limit>`. (you probably need just a single constant multiplier initially, either 1 or 10 or whatever we agree on).